groups/assemble/assemble_unsigned_psbt.js

```diff
@@ -114,16 +120,29 @@ module.exports = ({capacity, proposed, rate}, cbk) => {
 
           const funded = sumOf(member.utxos.map(n => n.witness_utxo.tokens));
 
-          return [
-            {
-              script: member.funding,
-              tokens: capacity,
-            },
-            {
-              script: member.change,
-              tokens: funded - committed(capacity, proposed) - (vbytes * rate),
-            },
-          ];
+          const fee = vbytes * rate;
+
+          // Calculate total output amount (including funding outputs)
+          const memberlength = !!member.funding ? (member.funding.length - 1) : 0;
```

Comment:
i'm not sure what's going on here, maybe the comments or code could be a bit more obvious like why it's subtracting 1 and sometimes the member length is zero